### DISCUSSION TITLE
PTT Command with Icom IC-7610

### DISCUSSION THREAD

**Discussion by f4emm** [foldynl/QLog#134] [Q&A]
Hello
I can't get the PTT control of my IC-7610 to work.
PTT State is enabled on the rig setting.
This command has no action :
ALT + \ PTT (must be enabled Rig Features: PTT State)
I was thinking of using Qlog as a remote station.
If anyone has the solution.
I thank you in advance.
Have a good day
F4EMM

**Comment by foldynl** [comment]
Hi Fabrice,

unfortunately, I cannot test it because I have only ic7300 but I would like to ask you. 

You wrote that PTT does not working and you are thinking to use it as remote Station.

PTT doesn't work for you when you try QLog as a remote station. Is it correct? How do you operate a remote station? Do you run Qlog on a remote desktop, or you have QLog locally and only connect RIGCTLd remotely? Still using Windows 8.1? 

the more details you give, the easier it will be to help you.

**Reply by f4emm** [reply]
Hello,
PC running Windows 10.
I use Icom remote utility to connect to the station via internet.
Icom remote utility creates a com port for me which I then use in Qlog.
In Qlog, I chose Hamlib IC-7610 stable / Port com from remote utility.
Everything works fine except the PTT function (Alt+\).
This solution works with Log4om but it is more resource intensive than Qlog.
I'm going to test again today, we'll see.
Thanks.
Have a good day.

**Comment by foldynl** [comment]
did you find any details?
